server/migrations/20190404145044-create-comments.js

```diff
@@ -0,0 +1,43 @@
+'use strict';
```

Comment:
Good job @davidokonji , you might consider removing this line to prevent some strict pattern in Javascript. or what do you think?

Reply:
Ok would work on that, thanks